helpers/utilities.php

```diff
@@ -17,6 +17,14 @@ function is_feature_enabled( $feature ) {
 	return FeatureFlags::get_instance()->flag_enabled( $feature );
 }
 
+function is_feature_flagged(  $feature) {
+	if ( ! empty( $feature ) || ! is_string( $feature ) ) {
```

Comment:
This check will return false even if the string is truthy.

Probably want to try:
`if ( empty( $feature ) || ! is_string( $feature ) ) {`